cmd/subnet-file-uploader.go

```diff
@@ -80,6 +87,10 @@ func (i *SubnetFileUploader) updateParams() {
 		i.filename += ".zst"
 		i.Params.Add("auto-compression", "zstd")
 	}
+	if i.AutoEncrypt || len(i.PubKey) >= 0 {
```

Comment:
```suggestion
	if i.AutoEncrypt || len(i.PubKey) > 0 {
```

This is causing the filename to become `<file>.enc.enc` in case of inspect.

Reply:
ok, is this used for inspect.... Yeah. That should be fixed, since it probably is also double encrypted.